beater/handlers.go

```diff
@@ -133,6 +137,55 @@ func isAuthorized(req *http.Request, secretToken string) bool {
 	return subtle.ConstantTimeCompare([]byte(parts[1]), []byte(secretToken)) == 1
 }
 
+func corsHandler(allowedOrigins []string, h http.Handler) http.Handler {
+
+	var isAllowed = func(origin string) bool {
+		for _, allowed := range allowedOrigins {
+			if origin == allowed {
```

Comment:
I'd do:

```
if allowed == "*" || allowed == origin {
    return true
}
```

that way, if someone adds a `*` at the end of the list, it will work as expected.

Reply:
and then 148 would just be: `return false`

Reply:
`http.example1.com, http.example2.com, *` is not valid semantics; it doesn't make sense. ACAO is either `*` (any) or an origin as described in spec:
`* | <origin>` 
and we extend it server side to `* | <origin ...>`

current implementation reflects that

Reply:
@jalvz this is not about the spec, this is about configuration. If you have a list of AllowedOrigin domains and then decide to add `*` because something isn't working as expected, or you want to temporarily allow all domains, it will not do what you expect.

Reply:
>  it will not do what you expect
it actually does
i know is configuration, what i meant is that our config *extends* the spec from `* or origin` to `* or list of origins`; as something like `origin1, *, origin2` doesn't make any sense to me. 
what users expect will depend on how we document it.

i changed it now.

does this PR need any further changes?